CharityDB/3 reports.sql

```diff
@@ -0,0 +1,24 @@
+--1. Which area (Postal Code) should I concentrate my efforts? I am not looking for the number of replies but the money that came in.
```

Comment:
Sorry, this probably requires some further explanation: In the UK the first part of the postcode is a reflection of the area, while the complete postcode is unique per few houses.  I want a report on the area, as opposed to a list of addresses.  Thanks in advance.

Reply:
fixed

Reply:
Thanks for this, this produces the correct result set, however there are postcodes with beginning 4 characters. Rather concentrate on the space.  Like this the postcode area NW11 is represented as NW1, which is not accurate.

Reply:
Oh sorry about that I thought I only saw ones with a max of 3...

Reply:
perfect thanks!